### PR TITLE
docs(store): add info for testing async actions

### PR DESCRIPTION
**Pull request by arturovt**
## PR Checklist
- [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Issue Number: #833

### CODE REVIEW

File: docs/recipes/unit-testing.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ describe('Zoo', () => {
     store = TestBed.get(Store);
   }));
 
-  it('it toggles feed', async(() => {
+  it('it toggles feed', () => {
```
Comment (arturovt) [inline docs/recipes/unit-testing.md]: Removed `async` as it's not necessary here.

**Comment by splincode** [comment]
Add also information about lifecycle actions

**Comment by arturovt** [comment]
@splincode it's not related to to unit testing

**Comment by splincode** [comment]
@arturovt ok

**Comment by arturovt** [comment]
@splincode I will create another PR for that but a bit later, actions lifecycle is an advanced topic, need more time

**Pull request merged**